testdata/project-v3-with-deploy-image/controllers/memcached_controller.go

```diff
@@ -24,20 +24,28 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
```suggestion
```

@NikhilSharmaWe we need to use the log from the controller and not this one.

Reply:
Yes, I was using that log only but I was using this for specifying the parameters of the finalize function but since now that parameter is not needed(due some changes in the finalize function), I am removing it.